src/test/java/hometaskstream/BookTest.java

```diff
@@ -0,0 +1,59 @@
+package hometaskstream;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class BookTest {
+
+    List<Book> bookList = List.of(new Book("abcde", 10), new Book("test", 12), new Book("java", 13));
+
+    @Test
+    void printAllBooks() {
+        System.out.println(Book.printAllBooks(Book.getRandomBooks(10)));
+    }
+
+    @Test
+    void printAllBooksName() {
+        System.out.println(Book.printAllBooksName(Book.getRandomBooks(10)));
+    }
+
+    @Test
+    void getPriceOfAllBooks() {
+        assertEquals(35, Book.getTotalPriceOfAllBooks(bookList));
```

Comment:
please add error messages to all assertions

Reply:
thanks, done